NuGet.config

```diff
@@ -2,8 +2,6 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
-    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
-    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <!-- Restore sources should be defined in build/sources.props. -->
```

Comment:
Is this file going to be standardized across repos?

Reply:
What is the point in keeping this file then?

Reply:
Yeah that was going to be my next question. Perhaps making sure that `<clear/>` is in effect?

Reply:
It doesn't. @natemcmaster had some explanation for this in the PR that we originally did this. @natemcmaster ?

Reply:
The purpose of keeping it around is mostly as a safety check. We want to make sure we're not leaking a dependency to some other feed. If a dev adds a new MSBuild project but the import to sources.props fails, we want restore to fail.